udata_hydra/migrations/__init__.py

```diff
@@ -11,7 +11,7 @@
 class Migrator:
     # NB: we can't use async __init__, hence the workaround
     @classmethod
-    async def create(cls, db_name, skip_errors=False):
+    async def create(cls, db_name, skip_errors=False) -> "Migrator":
```

Comment:
I understand you're using a string here as a forward reference to the type, but reading https://peps.python.org/pep-0649/ it feels that we can now simply use `Migrator` here? Maybe I'm mistaken, and it's not implemented yet?

Reply:
My type checker doesn't accept that, from what I understand https://peps.python.org/pep-0649/ has been accepted but will only be implemented in Python 3.14?